CMakeLists.txt

```diff
@@ -270,6 +270,30 @@ cxx_feature_check(STEADY_CLOCK)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+# If successful, the following variables will be defined:
```

Comment:
it might be cleaner to put this in a separate cmake file and call it as a function like `cxx_feature_check`. optional though.

Reply:
I can do that in a follow-up cl?

Reply:
sg